AUTHORS.rst

```diff
@@ -0,0 +1,9 @@
+The Box Python SDK is an open source project supported by `Box <https://box.com>`_
+used to interact with the Box API. This is a list of contributors.
+
+- `@Jeff-Meadows <https://github.com/Jeff-Meadows>`_
+- `@jmoldow <https://github.com/jmoldow>`_
+- `@aptxkid <https://github.com/aptxkid>`_
+- `@hnguyen08 <https://github.com/hnguyen08>`_
+- `@potrebic <https://github.com/potrebic>`_
+- `@nsundareswaran <https://github.com/nsundareswaran>`_
```

Comment:
We've had more contributors than just this. https://github.com/box/box-python-sdk/graphs/contributors

Also, should we also allow full names? I have a vague preference for my name in addition to, or instead of, my Github handle (but also still include the link to my profile).